docker-for-mac/faqs.md

```diff
@@ -175,11 +107,54 @@ VirtualBox or VMWare Fusion.
 
 ### What is the benefit of HyperKit?
 
-It is thinner than VirtualBox and VMWare fusion, and the version we include is
-tailor made for Docker workloads on the Mac.
+HyperKit is thinner than VirtualBox and VMWare fusion, and the version we include is tailor made for Docker workloads on the Mac.
```

Comment:
s/tailor made/tailor-made

Do you think this word could be confusing to non-native English speakers?

Reply:
Good point. I've updated to 'customized' for clarity.